go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/form3tech-oss/jwt-go v3.2.3+incompatible // indirect
 	github.com/go-chi/chi v4.0.2+incompatible
 	github.com/go-kit/kit v0.10.0 // indirect
-	github.com/go-piv/piv-go v1.7.0
+	github.com/go-piv/piv-go v1.9.1-0.20220211232127-7e03e7f49e54
```

Comment:
I've noticed you created the PR for supporting OpenBSD on `piv-go`. I will accept this version, but can you request `piv-go` maintainers to create a new release?

Reply:
Done: https://github.com/go-piv/piv-go/issues/105